### PR TITLE
Remove the array validation tests for the milestone award selector.

### PR DESCRIPTION
**Pull request by kberg**
This was more interesting a while ago, but the synergy list is on the verge of update with milestones and awards not related. The code that did this can be recovered and used if someone wants to provide a big update.

### CODE REVIEW

**Review by chosta — APPROVED** [review]
There is one commit that changes `SpaceElevator.ts`, please double check if you intended it for this PR.

File: src/cards/base/SpaceElevator.ts [hunk 1]
```diff
@@ -16,6 +17,7 @@ export class SpaceElevator extends Card implements IActionCard, IProjectCard {
       name: CardName.SPACE_ELEVATOR,
       tags: [Tags.SPACE, Tags.BUILDING],
       cost: 27,
+      productionDelta: Units.of({titanium: 1}),
```
Comment (vsrisuknimit) [inline src/cards/base/SpaceElevator.ts:20]: This is already fixed in #2468.

Comment (kberg) [reply src/cards/base/SpaceElevator.ts:20]: Dangit, I misused a branch.

**Comment by kberg** [comment]
A fresh pull should solve this.

**Pull request merged**